### PR TITLE
Move and merge two 5.11 migrations

### PR DESCRIPTION
**Pull request by MKodde**
These two migrations (Version20180910175453.php and Version20180910134145.php), where migrations to be run in 5.10. They have been dropped from that release and moved into the 5.11 release. This required them to be moved to a new migration with a new timestamp in order to allow running the migration without a problem.

Task 3 from https://www.pivotaltracker.com/story/show/166435147

### CODE REVIEW

File: database/DoctrineMigrations/Version20190504110100.php [hunk 1]
```diff
@@ -6,19 +6,27 @@
 use Doctrine\DBAL\Schema\Schema;
 
 /**
- * Auto-generated Migration: Please modify to your needs!
+ * This is a combined migration of the previous migration versions:
+ * - Version20180910134145
```
Comment (pablothedude) [inline database/DoctrineMigrations/Version20190504110100.php]: As a suggestion, maybe you could add the initial PR's?

File: database/DoctrineMigrations/Version20190504110100.php [hunk 2]
```diff
@@ -27,8 +35,9 @@ public function up(Schema $schema)
     public function down(Schema $schema)
     {
         // this down() migration is auto-generated, please modify it to your needs
```
Comment (pablothedude) [inline database/DoctrineMigrations/Version20190504110100.php]: How valuable is this? Because you effectively can't go back if the column data is lost.

Comment (thijskh) [reply database/DoctrineMigrations/Version20190504110100.php]: You can: should you downgrade to an older version: then just re push

File: database/DoctrineMigrations/Version20190504110100.php (hunk 1 above)
Comment (MKodde) [reply database/DoctrineMigrations/Version20190504110100.php]: Yes, good suggestion, added it to the docblock

**Pull request merged**